wandb/integration/keras/callbacks/metrics_logger.py

```diff
@@ -122,6 +122,8 @@ def on_batch_end(self, batch: int, logs: Optional[Dict[str, Any]] = None) -> Non
             wandb.log(logs)
 
             self.global_batch += self.log_freq
+
+        if self.log_freq == "batch" or isinstance(self.log_freq, int):
```

Comment:
I don't think we need `self.log_freq == "batch"` since `self.log_freq` is cast to `int` with value `1` in the constructor itself.